libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1283,6 +1289,26 @@ bool RC_Channel::run_aux_function(AUX_FUNC ch_option, AuxSwitchPos pos, AuxFuncT
     return ret;
 }
 
+#if AP_TIE_DOWN_CLAMPS_ENABLED
+void RC_Channel::do_aux_function_TIE_DOWN_RELEASE(const AuxSwitchPos ch_flag)
```

Comment:
How do you feel about putting "TIE_DOWN_RELEASE" in lower case?  It's a little shouty :-)